pages/login.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import KakaoLogin from '~/assets/svg/KakaoLoginButton.svg?component';
```

Comment:
assets/index.ts 에 모든 이미지들을 import 하고, 컴포넌트 단에서는 다음과 같이 한번에 가져오면 어떨까요? 
```suggestion
import {KakaoLogin, ThreePick, ThreePickLogo} from '~/assets';
```

Reply:
OMG such a smart 똑순 girl

수정하겠다.

Reply:
당신의 코멘트는 만듭니다. 많은 웃음.
이것은 입니까 레볼루션? 정말 완벽해!